import-export-cli/integration/testutils/testConstants.go

```diff
@@ -44,11 +44,16 @@ const DeploymentEnvYamlFilePath = "deployment_environments.yaml"
 const SampleAPIYamlFilePath = "testdata/sample-api.yaml"
 const SampleRevisionedAPIYamlFilePath = "testdata/sample-revisioned-api.yaml"
 const DevFirstUpdatedSampleCaseArtifactPath = "testdata/TestArtifactDirectory/DevFirstUpdatedSampleCaseArtifacts"
+const DevFirstHiddenFiles = "testdata/TestArtifactDirectory/HiddenFiles"
 const DevFirstUpdatedSampleCaseDocName = "/Docs/Doc1"
+const DevFirstUpdatedSampleCaseDocNameWithHiddenFiles = "/Docs/Doc2"
+const DevFirstUpdatedSampleCaseHiddenFilePathInRootDir = "/.hidden"
 const DevFirstUpdatedSampleCaseDocPath = DevFirstUpdatedSampleCaseArtifactPath + "/Doc1/testDoc.pdf"
+const DevFirstUpdatedSampleCaseHiddenFilePath = DevFirstHiddenFiles + "/.hidden"
```

Comment:
No usage for this constant. We can remove this too right?

Reply:
Fixed via [a227793](https://github.com/wso2/product-apim-tooling/pull/955/commits/a2277937391017a22de4d485b10d464ac9e9f7da)